### ISSUE TITLE
Infinite loop in Layout/ClosingParenthesisIndentation and Layout/SpaceInsideStringInterpolation

### ISSUE THREAD

**Issue by Linuus** [rubocop-hq/rubocop#7382]
When I run `rubocop -a` it crashes with `Infinite loop detected`. I guess this is because two cops are fighting each other?

--------

## Expected behavior

It should not end up in an infinite loop :) Not sure which cop should "win" though. How is this usually handled?

## Actual behavior

It finds itself in an infinite loop.

## Steps to reproduce the problem

Create the file:

```ruby
# frozen_string_literal: true

class Foo
  FOO = 'foo'
  BAR = /
    \A(foo)
    (, ?(#{
    FOO
  }))*\z
  /x.freeze
end
```

Run: `rubocop -a file_name.rb`

My output:

```
❯ rubocop -a rubocop-loop.rb
Inspecting 1 file
C

Offenses:

rubocop-loop.rb:9:1: C: [Corrected] Layout/ClosingParenthesisIndentation: Indent ) to column 2 (not 0)
}))*\z
^
rubocop-loop.rb:9:1: C: [Corrected] Layout/SpaceInsideStringInterpolation: Space inside string interpolation detected.
  }))*\z
^^

0 files inspected, 2 offenses detected, 2 offenses corrected
Infinite loop detected in /Users/linus/Desktop/rubocop-loop.rb.
/Users/linus/.asdf/installs/ruby/2.5.6/lib/ruby/gems/2.5.0/gems/rubocop-0.74.0/lib/rubocop/runner.rb:268:in `check_for_infinite_loop'
/Users/linus/.asdf/installs/ruby/2.5.6/lib/ruby/gems/2.5.0/gems/rubocop-0.74.0/lib/rubocop/runner.rb:251:in `block in iterate_until_no_changes'
/Users/linus/.asdf/installs/ruby/2.5.6/lib/ruby/gems/2.5.0/gems/rubocop-0.74.0/lib/rubocop/runner.rb:250:in `loop'
/Users/linus/.asdf/installs/ruby/2.5.6/lib/ruby/gems/2.5.0/gems/rubocop-0.74.0/lib/rubocop/runner.rb:250:in `iterate_until_no_changes'
/Users/linus/.asdf/installs/ruby/2.5.6/lib/ruby/gems/2.5.0/gems/rubocop-0.74.0/lib/rubocop/runner.rb:221:in `do_inspection_loop'
/Users/linus/.asdf/installs/ruby/2.5.6/lib/ruby/gems/2.5.0/gems/rubocop-0.74.0/lib/rubocop/runner.rb:124:in `block in file_offenses'
/Users/linus/.asdf/installs/ruby/2.5.6/lib/ruby/gems/2.5.0/gems/rubocop-0.74.0/lib/rubocop/runner.rb:142:in `file_offense_cache'
/Users/linus/.asdf/installs/ruby/2.5.6/lib/ruby/gems/2.5.0/gems/rubocop-0.74.0/lib/rubocop/runner.rb:122:in `file_offenses'
/Users/linus/.asdf/installs/ruby/2.5.6/lib/ruby/gems/2.5.0/gems/rubocop-0.74.0/lib/rubocop/runner.rb:110:in `process_file'
/Users/linus/.asdf/installs/ruby/2.5.6/lib/ruby/gems/2.5.0/gems/rubocop-0.74.0/lib/rubocop/runner.rb:87:in `block in each_inspected_file'
/Users/linus/.asdf/installs/ruby/2.5.6/lib/ruby/gems/2.5.0/gems/rubocop-0.74.0/lib/rubocop/runner.rb:86:in `each'
/Users/linus/.asdf/installs/ruby/2.5.6/lib/ruby/gems/2.5.0/gems/rubocop-0.74.0/lib/rubocop/runner.rb:86:in `reduce'
/Users/linus/.asdf/installs/ruby/2.5.6/lib/ruby/gems/2.5.0/gems/rubocop-0.74.0/lib/rubocop/runner.rb:86:in `each_inspected_file'
/Users/linus/.asdf/installs/ruby/2.5.6/lib/ruby/gems/2.5.0/gems/rubocop-0.74.0/lib/rubocop/runner.rb:73:in `inspect_files'
/Users/linus/.asdf/installs/ruby/2.5.6/lib/ruby/gems/2.5.0/gems/rubocop-0.74.0/lib/rubocop/runner.rb:39:in `run'
/Users/linus/.asdf/installs/ruby/2.5.6/lib/ruby/gems/2.5.0/gems/rubocop-0.74.0/lib/rubocop/cli.rb:210:in `execute_runner'
/Users/linus/.asdf/installs/ruby/2.5.6/lib/ruby/gems/2.5.0/gems/rubocop-0.74.0/lib/rubocop/cli.rb:80:in `execute_runners'
/Users/linus/.asdf/installs/ruby/2.5.6/lib/ruby/gems/2.5.0/gems/rubocop-0.74.0/lib/rubocop/cli.rb:51:in `run'
/Users/linus/.asdf/installs/ruby/2.5.6/lib/ruby/gems/2.5.0/gems/rubocop-0.74.0/exe/rubocop:13:in `block in <top (required)>'
/Users/linus/.asdf/installs/ruby/2.5.6/lib/ruby/2.5.0/benchmark.rb:308:in `realtime'
/Users/linus/.asdf/installs/ruby/2.5.6/lib/ruby/gems/2.5.0/gems/rubocop-0.74.0/exe/rubocop:12:in `<top (required)>'
/Users/linus/.asdf/installs/ruby/2.5.6/bin/rubocop:23:in `load'
/Users/linus/.asdf/installs/ruby/2.5.6/bin/rubocop:23:in `<main>'
```

## RuboCop version

```
❯ rubocop -V
0.74.0 (using Parser 2.6.4.1, running on ruby 2.5.6 x86_64-darwin18)
```

**Comment by buehmann** [comment]
This was already fixed in https://github.com/rubocop-hq/rubocop/pull/7255.

**Comment by Linuus** [comment]
Isn’t that fix in 0.74.0?

**Comment by buehmann** [comment]
No, the fix will be included in the next release (0.75.0, I guess).

**Comment by Linuus** [comment]
> No, the fix will be included in the next release (0.75.0, I guess).

You are correct!
Thanks!
